.github/workflows/format-code.yml

```diff
@@ -67,3 +71,17 @@ jobs:
           git add .
           git commit -m "[CI] Format code"
           git push
+      
+      - name: 'Post reminder to trigger build manually'
+        continue-on-error: true # Never fail the build if this fails
+        if: steps.fork_status.outputs.fork == 'true'
+        uses: actions/github-script@6e5ee1dc1cb3740e5e5e76ad668e3f526edbfe45 # 2.0.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'For security reasons, `/format` does not trigger CI builds when the PR has been submitted from a fork. To make the required checks pass, you need to trigger a build through one of the following ways:\n\n- Push an empty commit to this branch: `git commit --allow-empty -m "Trigger builds"`.\n- Close and reopen the PR.\n- Push a regular commit to this branch.'
```

Comment:
"security reasons" isn't really accurate but it's shorter than "to prevent workflows from triggering themselves and running into infinite loops" :P

Reply:
I like this.

Reply:
This is brilliant.